portfolio/src/main/webapp/js/project-detail.js

```diff
@@ -40,4 +40,39 @@ for(let detail of project.detail){
      Comment.populateAll(comments);
  }
 
+
+/** Methods to hide/show new comment form and related buttons. */
+class NewCommentForm {
+    /** @const */
+    static formNode = document.querySelector("#new-comment");
```

Comment:
I personally like seeing things like the const at the start of the line.  eg:
```
/** @const */ static formNode = ...
```

Reply:
Got it, I think I misread the style guide on this one. I'll clean these up throughout the repo in a patch.